### DISCUSSION TITLE
improve custom trained models

### DISCUSSION THREAD

**Discussion by DarkSoliditi** [explosion/spaCy#9357] [Help: Model Advice] [answered]
I have a model that's performing very well, but I a few edge cases that I need more training data for. Is there a way to continue training said model instead of having to train a brand new model with the complete dataset?

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @DarkSoliditi ! you can update an existing component with more examples for your edge cases. Check out the [NER Update Demo](https://github.com/explosion/projects/tree/v3/pipelines/ner_demo_update). You can also see more about [sourcing components](https://spacy.io/usage/processing-pipelines#sourced-components) and [resuming training](https://spacy.io/api/language#resume_training).
